### ISSUE TITLE
Accessibility discussion

### ISSUE THREAD

**Issue by jennaj** [galaxyproject/galaxy-hub#3630]
Discussion

I would like for us to build out a very clear how-to accessibility guide for users, then link this into more places. 

Assets

1.  Overview guide with the accessibility statement guide that includes "How" overview with links out to exact steps (or obvious link to another page with exact steps with enough graphic context it can be found with insufficient translation (someone is looking for a better way)
3. Link to guide at the bottom of the Hub with the other organizational-level links. The GTN does this -- the link is named "Accessibility"
4. As a distinct section in our the About ? page within Galaxy (refers back to Hub)
5. A guide at Help with how-to there then it refers back to Hub
6. Update server-specific pages at the Hub ex https://galaxyproject.org/us/
7. In a short movie for YouTube. This could be a blog post. 
8. A statement about who to contact about issues, contributions or potential future updates (our working list of knowns). Probably contact@
    * Internally, we have a point-person for triage intake or status questions. They maintain a simple user-facing summary list of requested/pending/deluxe updates somewhere public that can be shared. 

Components

1. Overview guide at the Hub (above)
2. "What is available?" type of overview
3. "How to set preferences ABC?" types of guides
4. Make sure these cover some practical how-to guides for different accessibility areas, such as setting a language preference.
5. Make sure users can find our guides easily, right when they need it

I think this all belongs at the Hub itself then is baked into Galaxy on About. Then GTN and Help have a short guide (found with a search) that explains accessibility how-to at those places then those link back to the Hub for how to do it in the Galaxy app. So, how-to is at each and all refer back to that master Hub page.

 _Even if some of this accessibility is maintained by different groups_ we can offer something simple to start from then add in more if needed for 1-off comms (see the example below). If it is software involved in a core Galaxy activity, I think we should try to capture it.  

Let's discuss please at GOATS. I know we have recent updates (https://github.com/galaxyproject/galaxy-hub/issues/3540) but I don't know the scope or status or existing plans. I was going to make a GTN FAQ for an item today then realized I didn't know exactly what we already have or what is planned (my fault!). Then I couldn't find the content link at the Hub for Galaxy itself, nor under About, but I did find the GTN content. Maybe we can make this nicer for users. 

Thanks! 

Use case

As an example for the top-level of "how", this is what I wrote out today at the forum in an the middle of an english discussion (no auto-translation, or not set within the user preferences) where I was later sharing a link to a non-english forum. It was focusing on language preferences because that was the most obvious accessibility tip needed for the specific use case -- a large guide about why and what might be available wouldn't cover this "how".

------
[details=" Translations and a forum's default language"]

:speaking_head: 

If a forum hosts discussions in a language other than one of yours, **no problem**!

Try logging into the forum. It probably has a language translation feature! 

Not interested in an account? Use a browser level language translation instead.

Example: [Google Chrome can apply an automatic translation from French to English](https://support.google.com/chrome/answer/173424). This allows an entire website to be navigated *even without an account* and all in your own language choice. You can write back in your language and the community will be able to read your messages. If they can't read your message for some reason, they will help you to get set up and possibly update their site to translate your language in the future.

This forum, Galaxy itself, and the GTN training site also offer **accessibility features**!
* **Help forum** -> Account -> Preferences -> Interface
* **Galaxy** -> User -> Preferences -> Manage Information -> Localization -> Preferred Language
* https://training.galaxyproject.org/training-material/accessibility.html
